__tests__/Column-tests.js

```diff
@@ -0,0 +1,636 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { Dimensions, StyleSheet } from 'react-native';
+import Column from '../src/Column';
+import { Provider as ThemeProvider } from '../src/Theme';
+
+let testDescription = '';
+let testData = '';
+
+// Test NOT Absolute, Case: 'lg'
+testDescription = 'Column should render View with {"width": "100%"}.\n';
+testData = 'Test Data: screen.type="lg" lg={12}';
+test(testDescription + testData, () => {
+  const screen = Dimensions.get('window');
+  screen.type = 'lg'; // Mocked screen.type. Hence, the following are ignored: screen.widght, screen.height
+
+  const tree = renderer.create(
+    <ThemeProvider>
+      <Column xs={9} sm={10} md={11} lg={12} screen={screen} />
+    </ThemeProvider>,
+  ).toJSON();
+  expect(tree).toMatchSnapshot();
+});
+
+// Test NOT Absolute, Case: 'lg'
+testDescription = 'Column should render View with {"width": "91.66666667%"}.\n';
+testData = 'Test Data: screen.type="lg" lg=null md={11}';
+test(testDescription + testData, () => {
```

Comment:
Is this format `testDescription` + `testData` a standard? Just curious why we are doing `test(testDescription + testData, () => {...})`

Reply:
Hi Gabriel, Good day! It is not a standard. Nonetheless, it is recommended to tell the purpose of a particular test. Hence, I have decided to input the test description and the test data. This is to easily read the snapshots and to easily identify if there's a need to update the snapshot when the tests are failing.

Reply:
Got it!